vuepress/docs/next/tutorials/getting-started/README.md

```diff
@@ -0,0 +1,372 @@
+---
```

Comment:
I think this duplicates the full page and assets which will make maintenance harder. I have another idea I'll send over on a new PR.